Svc/Deframer/docs/sdd.md

```diff
@@ -0,0 +1,58 @@
+\page SvcDeframer Deframer Component
+# Svc::Deframer Deframer Component
+
+Deframer takes a stream of bytes from ground and after checking the header and footer extracts data from the stream. The extracted data usually is passed to the service layer (typically CmdDisp, FileUplink, or a GenericHub). The Deframer uses a `DeframingProtocol` for the same abstraction that the Framer uses `FramingProtocol`. Deframer is backed by a circular buffer such that incomplete messages are reassembled. It also can be hooked up in two ways: 
```

Comment:
suggestions: add comma after `ground and` and after `the header` in the first sentence.